sw/device/tests/aon_timer_irq_test.c

```diff
@@ -100,6 +100,9 @@ static void execute_test(dif_aon_timer_t *aon_timer, uint64_t irq_time_us,
 
   uint32_t count_cycles =
       aon_timer_testutils_get_aon_cycles_from_us(irq_time_us);
+  // As AON_TIMER spends one more cycle to create the interrupt, subtract
+  // count_cycles by 1 here.
+  count_cycles = count_cycles - 1;
```

Comment:
As this is an aon_timer characteristic I'd prefer to change the `aon_timer_testutils_wakeup_config` function.

Reply:
make sense. I will move this inside the function.

Reply:
Just moved to actual start functions inside the dif not the testutil function. As you said this is AON_TIMER characteristic, so I think it is better to be in the DIF.